client-app/core/api/graphql/utils.ts

```diff
@@ -1,5 +1,30 @@
 import { intersection } from "lodash";
+import { ServerError } from "@/core/api/common";
+import { GraphQLErrorCode } from "@/core/api/graphql/enums";
 import type { ApolloClient } from "@apollo/client/core";
+import type { GraphQLErrors, NetworkError } from "@apollo/client/errors";
+import type { GraphQLError } from "graphql";
+
+export function hasErrorCode(graphQLErrors: ReadonlyArray<GraphQLError> | undefined, errorCode: GraphQLErrorCode) {
+  return graphQLErrors?.some((graphQLError) => graphQLError.extensions.code === errorCode);
+}
+
+export function toServerError(
```

Comment:
it is a function with the same name in client-app/core/api/common/utils/to-server-error.ts
you sure it should be two?

Reply:
Yes. They has different parameters. We probably may do overload, but they are weird style in typescript if they have different types for arguments and I'll prefer to keep everything for graphql in graphql folder.